examples/text_classification/run_script.sh

```diff
@@ -2,7 +2,7 @@ if [ ! -d ".data" ]; then
     mkdir .data
 fi
 
-python train.py AG_NEWS --device cuda --save-model-path  model.i --dictionary vocab.i
```

Comment:
@parmeet this might be helpful to keep the case where base-english tokenizer get used as well.